srcpkgs/i2pd/files/i2pd/log/run

```diff
@@ -0,0 +1 @@
+/usr/sbin/vlogger
```

Comment:
please align with how other `log/run` use `vlogger`

Reply:
It's a symlink, I thought thats how it's supposed to be these days? Did I miss something?

Reply:
oh i missed it's a symlink.
`fd --full-path 'log/run' -l ` reveals 25% of them are just symlinks to vlogger,  all others are scripts that `exec vlogger`

Reply:
symlink to /usr/bin/ not /usr/sbin

Reply:
I think we have it now. I'm sorry for the confusion. ;)